test/modules/load-gen/PriorityMessageSender.cs

```diff
@@ -97,17 +104,12 @@ public async override Task RunAsync(CancellationTokenSource cts, DateTime testSt
 
             this.Logger.LogInformation($"Sending finished. Now sending expected results to {Settings.Current.TestResultCoordinatorUrl}");
 
-            if (priorityAndSequence.Keys.Count <= 1)
-            {
-                throw new InvalidOperationException($"Must send more than 1 priority for valid test results. Priorities sent: {priorityAndSequence.Keys.Count}");
-            }
-
             // Sort priority by sequence number
             List<long> expectedSequenceNumberList = priorityAndSequence
                 .SelectMany(t => t.Value)
                 .ToList();
 
-            this.resultsSent = expectedSequenceNumberList.Count;
+            this.resultsSent = expectedSequenceNumberList.Count + 1;
```

Comment:
why add 1 here?

Reply:
To account for the first message, which must be sent and expected first regardless of its priority and TTL

Reply:
Please add a comment here.